root/entity/tags.tt

```diff
@@ -1,19 +0,0 @@
-[%- WRAPPER "$entity_type/layout.tt" title=l('Tags') page='tags' -%]
-    [%- INCLUDE "components/tags.tt" entity=entity -%]
-
-    [%- UNLESS display_genres.size OR display_tags.size OR c.user_exists -%]
-        <p>[% l('Nobody has tagged this yet.') %]</p>
-    [%- END -%]
-
-    [% IF c.user_exists %]
-        <script>
-          MB.init_main_tag_editor(
```

Comment:
TATBTLUO `init_main_tag_editor` so we can remove that plus `init_tag_editor` from root/static/scripts/common/components/TagEditor.js